GTG/plugins/send_email/sendEmail.py

```diff
@@ -32,22 +33,38 @@ class pluginSendEmail:
     """
     The plugin.
     """
+    def __init__(self):
+        """
+        Initialize variables.
+        """
+        self.browser_plugin_api = None
+        self.taskview_plugin_api = None
+        self.tb_Managerbutton = None
```

Comment:
Please, rename this variable to make it pep8 complient. no need to tb_ preffix. Let's say:
```python
self.main_button = None
```